joomla/joomla-cms
# [4.0] Group toolbar buttons for workflow stages and transitions

### Summary of Changes

This PR continues the grouping of action buttons in the backend (see #22832, #22393, #23337 and #23336, #23374, #23368, #23380). This time the workflow stages and transitions.

### Testing Instructions
Go to the workflows in the backend and check the toolbar. 
From there you can get stages and transitions. There should be a "Change status" dropdown now which activates when you select one or more workflows.

NOTE: A checkin button will be added as soon as the checked_out functionality is available



### Actual result



### Documentation Changes Required
